actionview/lib/action_view/renderer/partial_renderer.rb

```diff
@@ -403,7 +403,7 @@ def setup(context, options, block)
     def collection_from_options
       if @options.key?(:collection)
         collection = @options[:collection]
-        collection.respond_to?(:to_ary) ? collection.to_ary : []
+        collection.try!(:to_ary) ? collection.to_ary : []
```

Comment:
Yeah there are tests right now that support a nil collection, which is why i used try!